SAP2000_Engine/SAP2000_Engine.csproj

```diff
@@ -95,6 +95,11 @@
       <HintPath>C:\ProgramData\BHoM\Assemblies\Results_Engine.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="SAP2000_oM, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\..\ProgramData\BHoM\Assemblies\SAP2000_oM.dll</HintPath>
+      <Private>False</Private>
+    </Reference>
```

Comment:
This should be added as a project reference to avoid potential CI issues down the line :smile: